sentencepiece/Rakefile

```diff
@@ -78,5 +89,60 @@ class SentencePiecePackageTask < PackageTask
   end
 end
 
+class SentencePieceRepositoryTask < RepositoryTask
+  def initialize(packages_red_data_tools_repository, groonga_repository)
+    @packages_red_data_tools_repository = packages_red_data_tools_repository
+    @groonga_repository = groonga_repository
+  end
+
+  def define
+    super
+    Rake::Task["apt:upload"].clear
```

Comment:
Why do you replace `apt:upload`?

Reply:
Because keyring upload is not needed.
https://github.com/red-data-tools/packages.red-data-tools.org/blob/master/repository-task.rb#L458

Reply:
Is it harmful?
We already have keyring on packages.groonga.org: https://packages.groonga.org/debian/
It seems that managing our `apt:upload` task is costful than uploading keyring again.